docs/book/src/topics/managedcluster.md

```diff
@@ -12,9 +12,9 @@ custom resources:
 The combination of AzureManagedControlPlane/AzureManagedCluster
 corresponds to provisioning an AKS cluster. AzureManagedMachinePool
 corresponds one-to-one with AKS node pools. This also means that
-creating an AzureManagedControlPlane requires defining the default
-machine pool, since AKS requires at least one system pool at creation
-time.
+creating an AzureManagedCluster requires at least one AzureManagedMachinePool 
```

Comment:
probably leave this as AzureManagedControlPlane, since I have a PR open to delete AzureManagedCluster entirely

Reply:
ack